NtupleProducer/python/runPerformanceNTuple.py

```diff
@@ -102,6 +102,15 @@ def _add(name, what):
     copyUInts = cms.VInputTag(),
     copyFloats = cms.VInputTag(),
 )
+
+for X in ["tot","max"]:
+    for D in ['l1pfProducerBarrel', 'l1pfProducerHF', 'l1pfProducerHGCal']:
+        for I in "Calo EmCalo TK Mu".split(): 
+            process.ntuple.copyUInts.append( "%s:%sNL1%s" % (D,X,I))
+        for O in [""] + "Charged Neutral ChargedHadron NeutralHadron Photon Electron Muon".split():
+            process.ntuple.copyUInts.append( "%s:%sNL1PF%s" % (D,X,O))
+            process.ntuple.copyUInts.append( "%s:%sNL1Puppi%s" % (D,X,O))
+
```

Comment:
I think this should better be added within `monitorPerf()` so that if one runs multiple alternative PF versions (e.g. with different parameters) one can get the counters for those too.

Reply:
Moved to monitorPerf